ansible/roles/operator-pipeline/templates/openshift/tasks/verify-ci-results.yml

```diff
@@ -18,10 +19,14 @@ spec:
       workingDir: $(workspaces.results.path)
       script: |
         #! /usr/bin/env bash
-        # Did the tests pass?
+
         # did preflight/ CI have the latest version?
-        # If results exists, store them in the workspace
-        # In case of failure- comment on the PR with tailored support link
+        PREFLIGHT_VERSION=$(cat PREFLIGHT_RESULTS | jq '.test_library.version' | grep .)
+        if [$PREFLIGHT_VERSION < $(params.preflight_min_version)]; then
+          echo "preflight is having an outdated version $PREFLIGHT_VERSION"
+          echo -n "Mininum preflight version required is $(params.preflight_min_version)"
```

Comment:
no need for `-n` here, why would we trim next line in this case?

Reply:
Removed.